nss-day-cohort-41/nutshell-the-grave-and-the-rose
# Ba article2

Added post functionality to articles
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Type an article on the DOM and click the submit button.  Article should populate on DOM.
# Checklist:
